package.json

```diff
@@ -29,10 +29,13 @@
   "dependencies": {
     "bluebird": "^3.0.6",
     "expand-home-dir": "0.0.3",
-    "github": "^0.1.16",
+    "github": "^3.0.0",
```

Comment:
Required to access Enterprise github.
No need to specify a given version when instantiating the wrapper.